2015-08-06 02:28:25.815 [error] emulator Error in process <0.753.0> on node 'core@alex-N550JK' with exit value: {{nocatch,[{reason,{error,{invalid_object_member,109}}},{mfa,{ibot_webi_handler,websocket_info,3}},{stacktrace,[{jiffy,encode,2,[{file,"src/jiffy.erl"},{line,41}]},{ibot_webi_handler,websocket_info,3,[{file,"src/ibot_we... 


2015-08-06 02:28:25.817 [error] <0.541.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.753.0> exit with reason: {{nocatch,[{reason,{error,{invalid_object_member,109}}},{mfa,{ibot_webi_handler,websocket_info,3}},{stacktrace,[{jiffy,encode,2,[{file,"src/jiffy.erl"},{line,41}]},{ibot_webi_handler,websocket_info,3,[{file,"src/ibot_webi_handler.erl"},{line,149}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]},{msg,{<0.651.0>,{pubsub,wsuiclient},{send_data_to_ui,'BLA_BLA_BLA',{"message #0 for UI"}}}},{req,[{socket,#Port<0.941>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.753.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},48079}},{host,<<"localhost">>},{host_info,undefined},{port,10101},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10101">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10101">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"CW5hgC1fxMh+R5UY8MKQ0g==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}]},[{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]}

2015-08-06 02:33:24.399 [error] emulator Error in process <0.753.0> on node 'core@alex-N550JK' with exit value: {[{reason,badarith},{mfa,{ibot_webi_handler,websocket_info,3}},{stacktrace,[{ibot_webi_handler,websocket_info,3,[{file,"src/ibot_webi_handler.erl"},{line,149}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_we... 


2015-08-06 02:33:24.400 [error] <0.541.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.753.0> exit with reason: {[{reason,badarith},{mfa,{ibot_webi_handler,websocket_info,3}},{stacktrace,[{ibot_webi_handler,websocket_info,3,[{file,"src/ibot_webi_handler.erl"},{line,149}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]},{msg,{<0.651.0>,{pubsub,wsuiclient},{send_data_to_ui,'BLA_BLA_BLA',{"message #0 for UI"}}}},{req,[{socket,#Port<0.941>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.753.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},48137}},{host,<<"localhost">>},{host_info,undefined},{port,10101},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10101">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10101">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"3OPbulz9Hm8fn9XPFslg2w==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}],[{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]}

2015-08-06 02:37:14.383 [error] emulator Error in process <0.754.0> on node 'core@alex-N550JK' with exit value: {{nocatch,[{reason,{error,{invalid_object_member,116}}},{mfa,{ibot_webi_handler,websocket_info,3}},{stacktrace,[{jiffy,encode,2,[{file,"src/jiffy.erl"},{line,41}]},{ibot_webi_handler,websocket_info,3,[{file,"src/ibot_we... 


2015-08-06 02:37:14.384 [error] <0.541.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.754.0> exit with reason: {{nocatch,[{reason,{error,{invalid_object_member,116}}},{mfa,{ibot_webi_handler,websocket_info,3}},{stacktrace,[{jiffy,encode,2,[{file,"src/jiffy.erl"},{line,41}]},{ibot_webi_handler,websocket_info,3,[{file,"src/ibot_webi_handler.erl"},{line,149}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]},{msg,{<0.651.0>,{pubsub,wsuiclient},{send_data_to_ui,'BLA_BLA_BLA',{"message #0 for UI"}}}},{req,[{socket,#Port<0.942>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.754.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},48183}},{host,<<"localhost">>},{host_info,undefined},{port,10101},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10101">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10101">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"PR6MxLxOVMZmSUaiDZdfTA==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}]},[{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]}

2015-08-06 02:38:44.601 [error] emulator Error in process <0.756.0> on node 'core@alex-N550JK' with exit value: {{try_clause,ok},[{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]}


2015-08-06 02:38:44.602 [error] <0.542.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.756.0> exit with reason: {{try_clause,ok},[{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]}

2015-08-20 03:13:13.060 [error] <0.650.0> gen_server ibot_nodes_srv_connector terminated with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:call/2 line 182
2015-08-20 03:13:13.060 [error] <0.650.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:terminate/7 line 804
2015-08-20 03:13:13.061 [error] <0.760.0> gen_server test_py_monitor terminated with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}}
2015-08-20 03:13:13.062 [error] <0.760.0> CRASH REPORT Process test_py_monitor with 0 neighbours exited with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:terminate/7 line 804
2015-08-20 03:13:13.062 [error] <0.763.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}}
2015-08-20 03:13:13.062 [error] <0.763.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:terminate/7 line 804
2015-08-20 03:13:13.062 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.650.0> exit with reason {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in context child_terminated
2015-08-20 03:13:15.285 [error] <0.784.0> gen_server ibot_nodes_srv_connector terminated with reason: {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in gen_server:call/2 line 182
2015-08-20 03:13:15.286 [error] <0.784.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in gen_server:terminate/7 line 804
2015-08-20 03:13:15.287 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.784.0> exit with reason {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in context child_terminated
